src/fsharp/vs/service.fs

```diff
@@ -1634,151 +1634,154 @@ module internal Parser =
-
-                let hashLoadBackgroundDiagnostics, otherBackgroundDiagnostics = 
-                    backgroundDiagnostics 
-                    |> List.partition (fun backgroundError -> 
-                        hashLoadsInFile 
-                        |>  List.exists (fst >> sameFile (fileOfBackgroundError backgroundError)))
-
-                // Create single errors for the #load-ed files.
-                // Group errors and warnings by file name.
-                let hashLoadBackgroundDiagnosticsGroupedByFileName = 
-                    hashLoadBackgroundDiagnostics 
-                    |> List.map(fun err -> fileOfBackgroundError err,err) 
-                    |> List.groupByFirst  // fileWithErrors, error list
-
-                //  Join the sets and report errors. 
-                //  It is by-design that these messages are only present in the language service. A true build would report the errors at their
-                //  spots in the individual source files.
-                for (fileOfHashLoad, rangesOfHashLoad) in hashLoadsInFile do
-                    for errorGroupedByFileName in hashLoadBackgroundDiagnosticsGroupedByFileName do
-                        if sameFile (fst errorGroupedByFileName) fileOfHashLoad then
-                            for rangeOfHashLoad in rangesOfHashLoad do // Handle the case of two #loads of the same file
-                                let diagnostics = snd errorGroupedByFileName |> List.map(fun (pe,f)->pe.Exception,f) // Strip the build phase here. It will be replaced, in total, with TypeCheck
-                                let errors = [ for (err,sev) in diagnostics do if sev = FSharpErrorSeverity.Error then yield err ]
-                                let warnings = [ for (err,sev) in diagnostics do if sev = FSharpErrorSeverity.Warning then yield err ]
-                                
-                                let message = HashLoadedSourceHasIssues(warnings,errors,rangeOfHashLoad)
-                                if errors=[] then warning(message)
-                                else errorR(message)
-
-                // Replay other background errors.
-                for (phasedError,sev) in otherBackgroundDiagnostics do
-                    if sev = FSharpErrorSeverity.Warning then 
-                        warning phasedError.Exception 
-                    else errorR phasedError.Exception
-
-            | None -> 
-                // For non-scripts, check for disallow #r and #load.
-                ApplyMetaCommandsFromInputToTcConfig tcConfig (parsedMainInput,Path.GetDirectoryName mainInputFileName) |> ignore
-                
-            // A problem arises with nice name generation, which really should only 
-            // be done in the backend, but is also done in the typechecker for better or worse. 
-            // If we don't do this the NNG accumulates data and we get a memory leak. 
-            tcState.NiceNameGenerator.Reset()
+        async {
```

Comment:
Could you please add `?w=1` into the address to make diffs ignore whitespace differences instead? 

https://github.com/Microsoft/visualfsharp/pull/2315/files?w=1